--- origsrc/fox-1.6.51/doc/Makefile.am	2014-05-13 23:32:40.000000000 -0500
+++ src/fox-1.6.51/doc/Makefile.am	2016-08-09 20:43:15.561885500 -0500
@@ -60,6 +60,7 @@ xml.html
 
 
 docs:
+	test -f doxygen.cfg || $(SED) -e "s|\.\./|$(top_srcdir)/|g" $(srcdir)/doxygen.cfg > doxygen.cfg
 	doxygen doxygen.cfg
 
 
--- origsrc/fox-1.6.51/doc/doxygen.cfg	2014-05-13 23:32:40.000000000 -0500
+++ src/fox-1.6.51/doc/doxygen.cfg	2016-08-09 21:12:00.434699800 -0500
@@ -485,7 +485,7 @@ IMAGE_PATH             =
 # input file. Doxygen will then use the output that the filter program writes
 # to standard output.
 
-INPUT_FILTER           = ./filter.pl
+INPUT_FILTER           = ../doc/filter.pl
 
 # If the FILTER_SOURCE_FILES tag is set to YES, the input filter (if set using
 # INPUT_FILTER) will be used to filter the input files when producing source
@@ -581,13 +581,13 @@ HTML_FILE_EXTENSION    = .html
 # each generated HTML page. If it is left blank doxygen will generate a
 # standard header.
 
-HTML_HEADER            = header.html
+HTML_HEADER            = ../doc/header.html
 
 # The HTML_FOOTER tag can be used to specify a personal HTML footer for
 # each generated HTML page. If it is left blank doxygen will generate a
 # standard footer.
 
-HTML_FOOTER            = footer.html
+HTML_FOOTER            = ../doc/footer.html
 
 # The HTML_STYLESHEET tag can be used to specify a user-defined cascading
 # style sheet that is used by each HTML page. It can be used to
@@ -596,7 +596,7 @@ HTML_FOOTER            = footer.html
 # the style sheet file to the HTML output directory, so don't put your own
 # stylesheet in the HTML output directory as well, or it will be erased!
 
-HTML_STYLESHEET        = page.css
+HTML_STYLESHEET        = ../doc/page.css
 
 # If the HTML_ALIGN_MEMBERS tag is set to YES, the members of classes,
 # files or namespaces will be aligned in HTML using tables. If set to
